sdk/python/tests/compiler/testdata/katib.log

```diff
@@ -0,0 +1,20 @@
+[mnist-hpo : main] INFO:root:Generating experiment template.
+[mnist-hpo : main] INFO:root:Creating kubeflow.org/experiments mnist in namespace kubeflow.
+[mnist-hpo : main] ERROR:root:Exception when create kubeflow.org/experiments: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"experiments.kubeflow.org is forbidden: User \"system:serviceaccount:kubeflow:default\" cannot create resource \"experiments\" in API group \"kubeflow.org\" in the namespace \"kubeflow\"","reason":"Forbidden","details":{"group":"kubeflow.org","kind":"experiments"},"code":403}
+[mnist-hpo : main] 
+[mnist-hpo : main] Traceback (most recent call last):
+[mnist-hpo : main]   File "/ml/launch_experiment.py", line 137, in <module>
+[mnist-hpo : main]     main()
+[mnist-hpo : main]   File "/ml/launch_experiment.py", line 120, in main
+[mnist-hpo : main]     create_response = experiment.create(inst)
+[mnist-hpo : main]   File "/ml/launch_crd.py", line 98, in create
+[mnist-hpo : main]     self._log_and_raise_exception(e, "create")
+[mnist-hpo : main]   File "/ml/launch_crd.py", line 135, in _log_and_raise_exception
+[mnist-hpo : main]     raise ex
+[mnist-hpo : main] kubernetes.client.rest.ApiException: (403)
+[mnist-hpo : main] Reason: Forbidden
+[mnist-hpo : main] HTTP response headers: HTTPHeaderDict({'Date': 'Tue, 09 Jun 2020 02:46:29 GMT', 'Content-Length': '355', 'Content-Type': 'application/json', 'X-Content-Type-Options': 'nosniff', 'Cache-Control': 'no-cache, private'})
+[mnist-hpo : main] HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"experiments.kubeflow.org is forbidden: User \"system:serviceaccount:kubeflow:default\" cannot create resource \"experiments\" in API group \"kubeflow.org\" in the namespace \"kubeflow\"","reason":"Forbidden","details":{"group":"kubeflow.org","kind":"experiments"},"code":403}
```

Comment:
For this your namespace needs katib permission. If we are going to do integration test with this we need to follow these requirements. https://github.com/kubeflow/kfp-tekton/tree/master/samples/katib

The katib unit test here was for checking the katib variables is outputted properly.

Reply:
The katib test also takes a while because it's doing 9 trail runs for the test.

Reply:
I added this one to the ignored tests for now